tests/unit/test_grid.py

```diff
@@ -493,3 +469,65 @@ def test_dims_are_projected_x_y(
     file_fixture = request.getfixturevalue(file_fixture_name)
     var_info = var_info_fxn(file_fixture)
     assert dims_are_projected_x_y(dimensions, var_info) is expected_result
+
+
+@pytest.mark.parametrize(
+    'test_extent, expected, description',
+    [
+        ((1, 2, 3, 4), (1, 2, 3, 4), 'no reordering necessary'),
+        ((3, 2, 1, 4), (1, 2, 3, 4), 'x needs swapped'),
+        ((1, 4, 3, 2), (1, 2, 3, 4), 'y needs swapped'),
+        ((3, 4, 1, 2), (1, 2, 3, 4), 'x  and y need swapped'),
+    ],
+)
+def test_reorder_extents(test_extent, expected, description):
+    assert expected == reorder_extents(*test_extent)
+
+
+@patch('harmony_regridding_service.grid.create_area_def', wraps=create_area_def)
+def test_convert_projected_area_to_geographic_ease_grid(mock_create_area_def):
+    # Create a projected AreaDefinition using EPSG:6933 (NSIDC EASE-Grid 2.0
+    # Global) Reversing the area extent values for good measure.
```

Comment:
Super nit: Couldn't this just be a documentation string? :disappear:

Reply:
f284a31